package.json

```diff
@@ -6,6 +6,13 @@
         "url": "https://github.com/dhis2/ui.git"
     },
     "author": "Viktor Varland <viktor@dhis2.org>",
+    "contributors": [
+        {
+            "name": "Jan-Gerke Salomon",
+            "email": "jan-gerke@dhis2.org",
+            "url": "http://functionalprogramming.ninja"
+        }
+    ],
```

Comment:
:sunglasses:

Reply:
Should we also add the `"awesome": "yes"` field here?

Reply:
Haha, good idea! Jokes aside, I could add all contributors to this field, using their dhis2 address?

Reply:
For me personally that's not necessary. But I don't know how the others feel.

Reply:
Not necessary or not wanted?

Reply:
Well, I don't really have a strong opinion on it, but I feel like the entire organisation owns this library. Having a couple people as the designated contributors maybe communicates a conflicting message. But as I said, I don't really have too strong of an opinion on it.